source/content/nutzen/api-nutzen.rst

```diff
@@ -1,154 +1,473 @@
+    Das Portal `opendata.swiss <https://opendata.swiss/>`__ basiert auf dem Open-Source-Projekt CKAN. 
+    CKAN stellt eine umfangreiche API für die Metadaten des offenen Datenkatalogs zur Verfügung, die Applikationsentwickler_innen nutzen können. 
+    In diesem Kapitel erklären wir Ihnen, wie Sie mittels API auf die Daten von opendata.swiss zugreifen können. Werfen Sie auch einen Blick 
+    in aktuelle `Beispiele, wie OGD genutzt <https://opendata.swiss/showcase/>`__ wird.
 
-Nutzung der API – so funktioniert’s
+.. admonition:: Gut zu wissen
+
+    Hier geben wir Ihnen eine Einführung zur Nutzung der API von `opendata.swiss <https://opendata.swiss/>`__. 
+    Eine ausführliche Dokumentation zur CKAN API finden Sie in der `CKAN API Dokumentation <https://docs.ckan.org/en/latest/api/>`__.
+
+
+.. _api-nutzen-einführung:
+
+Einführung
+^^^^^^^^^^
+
+Eine `API (Application Programming Interface <https://de.wikipedia.org/wiki/Programmierschnittstelle>`__) ermöglicht es, dass Software direkt mit einem externen System (in diesem Fall den Metadaten auf der Plattform opendata.swiss) interagiert. Während die Webseite der Plattform opendata.swiss und die `Suchfunktion <https://opendata.swiss/de/dataset>`__ auf die Nutzung durch Menschen ausgerichtet ist, bieten wir über die API eine Zugriffsmöglichkeit, die für die Automatisierung optimiert ist.
+
+
+Nutzung der API – so funktioniert es
 ------------------------------------
 
 - :ref:`Zugriffsmöglichkeiten <api-nutzen-zugriffsmoeglichkeiten>`
 - :ref:`Action-API <api-nutzen-action-api>`
+- :ref:`Beispiel API-Response <api-nutzen-beispiel-response>`
 - :ref:`Terminologie <api-nutzen-terminologie>`
 - :ref:`Beispiele <api-nutzen-beispiele>`
 
-.. admonition:: Gut zu wissen
 
-   Hier geben wir Ihnen eine kompakte Einführung zur Nutzung der API
-   von opendata.swiss. Eine ausführliche Dokumentation zur CKAN API
-   finden Sie in der
-   `CKAN API Dokumentation <https://docs.ckan.org/en/latest/api/>`__.
 
 .. _api-nutzen-zugriffsmoeglichkeiten:
 
 Zugriffsmöglichkeiten
 ^^^^^^^^^^^^^^^^^^^^^
 
-Es gibt 3 Zugriffsarten, die Ihnen bei opendata.swiss zur Verfügung stehen:
+Sie haben verschiedene Möglichkeiten, auf die API von opendata.swiss zuzugreifen: 
 
-.. code-block::
-   :caption: API Zugriff über den Browser
+- via Browser, 
+- über die Kommandozeile, oder durch 
+- spezialisierte Werkzeuge wie ckanapi oder Fetch-API. 
 
-   https://opendata.swiss/api/3/action/package_search?fq=tags:economy
+Je nach Einsatzzweck eignen sich verschiedene Arten. Für den Zugriff über den Browser empfehlen wir die Nutzung eines JSON-Viewers 
+(nativ oder Plug-In), um die Lesbarkeit der Informationen zu verbessern.
 
-.. code-block::
-   :caption: API Zugriffe über die Kommandozeile, etwa mit Curl
+Die Basis-URL lautet https://ckan.opendata.swiss/api/3/action/ und wird gefolgt von der Aktion und allfälligen notwendigen Zusatzinformationen.
```

Comment:
do you want to have `https://ckan.opendata.swiss/api/3/action/` as a link?
because people will get "404 Not Found" while following that link